### PR TITLE
Added info for Inkscape RC 1.0

### PR DESCRIPTION
**Pull request by SeanLeftBelow**
Adding a few notes from my Inkscape RC 1.0 install. 
Tried to push changes via new branch and gh-pages branch per #121 but I don't have permissions sorted completely yet? 
I am more than happy to add changes via forked repo/branch PR.

### CODE REVIEW

File: collections/_docs/en/001-001-install.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ For the installation process watch the video for your operating system
 
 ## Requirements
 
-* [Inkscape](https://inkscape.org/) Version 0.92.2 or higher
+* [Inkscape](https://inkscape.org/) Version 0.92.2 and [Inkscape](https://inkscape.org/release/inkscape-1.0/?latest=1) Version 1.0 Release Candidate.
```
Comment (kaalleen) [inline collections/_docs/en/001-001-install.md:19]: Maybe we should say Inkscape Version 0.92.2 or higher (including Inkscape Version 1.0 [...])

Current version is 0.92.5, we don't want to drop support on this one ;)

**Comment by kaalleen** [comment]
Thanks for your additional comments to the install process. I think this really will help others.
I left just one single comment on the changes. What do you think?

Hm, looks like we have to check your permissions again.
Thought you'd be able to push to this repository?!

File: collections/_docs/en/001-001-install.md (hunk 1 above)
Comment (SeanLeftBelow) [reply collections/_docs/en/001-001-install.md]: Ahh, yes, or else it implies it doesn't work with 0.92+
Fixed.

**Comment by SeanLeftBelow** [comment]
@kaalleen I thought this as well, but my pushes were responded back with 403's. No biggie, I want to keep contributing and I can do so this way.

**Comment by kaalleen** [comment]
Thanks, I've merged it into the gh-pages, should be on there very soon!

@lexelby could you have a look why @SeanLeftBelow isn't able to push (when you find the time)?

**Comment by lexelby** [comment]
@SeanLeftBelow No idea why you're getting 403... is it an SSH key issue?  We have you marked as having Write access on this repo, so it should just work.

**Pull request merged**